scripts/apidocgen/generate.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# This script generates swagger description file and required Go docs files
+# from the coderd API.
+
+set -euo pipefail
```

Comment:
should source lib.sh (see other scripts for examples) and use `log` instead of `echo`

Reply:
Fixed